zellij-server/src/tab.rs

```diff
@@ -259,7 +259,11 @@ impl Tab {
             index,
             position,
             panes,
-            name,
+            name: if name.is_empty() {
```

Comment:
I would prefer the initialization here just like the `panes` are initialized.

`let name =` prior and then keep the name here.